editor/src/components/canvas/canvas-strategies/strategies/group-conversion-helpers.ts

```diff
@@ -534,41 +599,80 @@ export function convertFrameToFragmentCommands(
 
   // if any children is not position: absolute, bail out from the conversion
   if (childInstances.some((child) => MetadataUtils.elementParticipatesInAutoLayout(child))) {
+    return conversionForbidden('Frame children must be positioned absolutely')
+  }
+
+  return { element: instance.element.value, childInstances: childInstances }
+}
+
+export function convertFrameToFragmentCommands(
+  metadata: ElementInstanceMetadataMap,
+  pathTrees: ElementPathTrees,
+  allElementProps: AllElementProps,
+  elementPath: ElementPath,
+): Array<CanvasCommand> {
+  const instance = getInstanceForFrameToFragmentConversion(
+    metadata,
+    pathTrees,
+    allElementProps,
+    elementPath,
+  )
+  if (isConversionForbidden(instance)) {
     return []
   }
 
-  const { children, uid } = instance.element.value
+  const { children, uid } = instance.element
 
   const parentOffset =
     MetadataUtils.findElementByElementPath(metadata, elementPath)?.specialSizeMeasurements.offset ??
     zeroCanvasPoint
 
   return [
     deleteElement('always', elementPath),
-    addElement('always', childInsertionPath(parentPath), jsxFragment(uid, children, true), {
-      indexPosition: absolute(MetadataUtils.getIndexInParent(metadata, pathTrees, elementPath)),
-      importsToAdd: {
-        react: {
-          importedAs: 'React',
-          importedFromWithin: [],
-          importedWithName: null,
+    addElement(
+      'always',
+      childInsertionPath(EP.parentPath(elementPath)),
+      jsxFragment(uid, children, true),
+      {
+        indexPosition: absolute(MetadataUtils.getIndexInParent(metadata, pathTrees, elementPath)),
+        importsToAdd: {
+          react: {
+            importedAs: 'React',
+            importedFromWithin: [],
+            importedWithName: null,
+          },
         },
       },
-    }),
-    ...offsetChildrenByVectorCommands(childInstances, parentOffset),
+    ),
+    ...offsetChildrenByVectorCommands(instance.childInstances, parentOffset),
   ]
 }
 
-export function convertFrameToGroup(
+type ConversionForbidden = {
+  forbidden: true
```

Comment:
Typically we'd use:
```ts
type: 'CONVERSION_FORBIDDEN`
```
for objects / types that we want to write a `is_()` type guard for

Reply:
[`253c2f2` (#4315)](https://github.com/concrete-utopia/utopia/pull/4315/commits/253c2f2ec519021a10d942f6fac27d963497f850)